ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -15,7 +15,6 @@ type ${project.artifactId}_config =  {
      'mode' : long = 0644
      'owner' : string = 'root'
      'group' : string = 'root'
-     'daemon' ? string[]
```

Comment:
how nicely backwards incompatible.
for migration purposes you could use 
```
'daemon' ? string with error("use daemons")
```
did we agree on backwards incompatible changes for this issue?